pyiron_atomistics/vasp/base.py

```diff
@@ -53,6 +53,17 @@
 __status__ = "production"
 __date__ = "Sep 1, 2017"
 
+def _vasp_generic_energy_free_affected(job):
+    """
+    Checks whether the value saved in output/generic/energy_pot matches the electronic free energy.
+    """
+    if self.project_hdf5.get("HDF_VERSION", "0.1.0") == "0.1.0":
+        energy_free = np.array([e[-1] for e in self.project_hdf5["output/generic/dft/scf_energy_pot"]])
+        energy_pot = self.project_hdf5["output/generic/energy_pot"]
+        return not np.allclose(energy_free, energy_pot)
```

Comment:
Isn't it other way around? Shouldn't it be `True` when they are all close?

Reply:
Oh, it needs to be scf_energy_free.